### DISCUSSION TITLE
How to use raw values as current values?

### DISCUSSION THREAD

**Discussion by mscottwvc** [NightscoutFoundation/xDrip#3056] [Q&A]
Using juggluco with libre 3 to pass values into xdrip.  I can see each minute value come in as a raw value, but xdrip just shows the current value every 5 minutes.  How do I get xdrip to use each raw value as the current value (main goal is use raw values to trigger alarms)?

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
That's still an engineering mode feature.
